rust-src/wallet_library/src/test_helpers.rs

```diff
@@ -0,0 +1,63 @@
+use concordium_base::{
+    common::Versioned,
+    id::{
+        constants::{self, AttributeKind},
+        types::{ArIdentity, ArInfo, GlobalContext, IdentityObjectV1, IpInfo},
+    },
+};
+use std::{collections::BTreeMap, fs, path::PathBuf};
+
+#[cfg(test)]
+fn base_path() -> String {
```

Comment:
Paths should usually be `PathBuf`, so it's unclear why we have this complicated function. Wouldn't

```
fn base_path() -> PathBuf {
     Path::new(env!("CARGO_MANIFEST_DIR")).join("resources")
}
```
be the thing we want?

Reply:
Yes, I just thought I needed the String. I have changed it as suggested.